examples/multi_window_ref_app/lib/app/popup_window_content.dart

```diff
@@ -0,0 +1,79 @@
+import 'package:flutter/material.dart';
+import 'package:multi_window_ref_app/app/child_window_renderer.dart';
+import 'package:multi_window_ref_app/app/positioner_settings.dart';
+import 'package:multi_window_ref_app/app/window_controller_text.dart';
+import 'window_settings.dart';
+import 'window_manager_model.dart';
+
+class PopupWindowContent extends StatelessWidget {
+  const PopupWindowContent(
+      {super.key,
+      required this.controller,
+      required this.windowSettings,
+      required this.positionerSettingsModifier,
+      required this.windowManagerModel});
+
+  final PopupWindowController controller;
+  final WindowSettings windowSettings;
+  final PositionerSettingsModifier positionerSettingsModifier;
+  final WindowManagerModel windowManagerModel;
+
+  @override
+  Widget build(BuildContext context) {
+    return ViewAnchor(
+        view: ChildWindowRenderer(
+            windowManagerModel: windowManagerModel,
+            windowSettings: windowSettings,
+            positionerSettingsModifier: positionerSettingsModifier,
+            controller: controller),
+        child: Scaffold(
+            backgroundColor: Colors.transparent,
+            body: SingleChildScrollView(
+                child: Center(
```

Comment:
I think `SingleChildScrollView` should be removed. It prevents the popup background from filling the widget.
`Center` could be removed too, as it doesn't seem to have any effect when the popup fills the view.

Reply:
I am using that or else the window will overflow and not be able to scroll. Do we wan that behavior? I would assume we want scrolling there

Reply:
But ill also fix that problem that you're pointing out

Reply:
Fixed! It scrolls and expands appropriately now